Source/Core/HermiteSpline.js

```diff
@@ -574,6 +609,10 @@ HermiteSpline.prototype.evaluate = function (time, result) {
     timeVec
   );
 
+  // Multiply the in-tangent and out-tangent values by the time delta.
+  coefs.z *= timesDelta;
+  coefs.w *= timesDelta;
```

Comment:
According to the [cubic interpolation appendix](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#appendix-c-interpolation) of the glTF spec, the formula for cubic interpolation between keyframes is as follows:
![image](https://user-images.githubusercontent.com/32226860/164767880-ba2a94d4-0f3c-41a9-8e14-6dc033bda84a.png)

Here, <img src="https://render.githubusercontent.com/render/math?math=a_k, v_k, b_k"> refer to the in-tangent, the property value, and the out-tangent of the `k`th keyframe. The value `t` is the current requested timestamp, and <img src="https://render.githubusercontent.com/render/math?math=t_d = t_{k+1} - t_k"> refers to the delta between the keyframed times, i.e. the duration of the interpolated segment.

This last part, <img src="https://render.githubusercontent.com/render/math?math=t_d = t_{k+1} - t_k">, is what wasn't being factored into the Hermite equation. This can be left out for interpolation between keyframes that are exactly 1.0 apart, but the class shouldn't make this assumption.

Reply:
Thanks for explaining! though could somewhere in this function we document which coefficient is which of the 4 components? is it `(start, end, in tangent, out tangent)` or some other order?